src/main/java/org/codehaus/mojo/flatten/FlattenMojo.java

```diff
@@ -657,18 +657,39 @@ protected Model createCleanPom( Model effectivePom ) throws MojoExecutionExcepti
         }
 
         // transform profiles...
+        Dependencies managedDependencies = new Dependencies();
+        if (effectivePom.getDependencyManagement() != null
+                && effectivePom.getDependencyManagement().getDependencies() != null)
+            managedDependencies.addAll(effectivePom.getDependencyManagement().getDependencies());
+
         for ( Profile profile : effectivePom.getProfiles() )
         {
             if ( !isEmbedBuildProfileDependencies() || !isBuildTimeDriven( profile.getActivation() ) )
             {
                 if ( !isEmpty( profile.getDependencies() ) || !isEmpty( profile.getRepositories() ) )
                 {
-                    Profile strippedProfile = new Profile();
-                    strippedProfile.setId( profile.getId() );
-                    strippedProfile.setActivation( profile.getActivation() );
-                    strippedProfile.setDependencies( profile.getDependencies() );
-                    strippedProfile.setRepositories( profile.getRepositories() );
-                    cleanPom.addProfile( strippedProfile );
+                    List<Dependency> strippedDependencies = new ArrayList<>();
+                    for (Dependency dep : profile.getDependencies()) {
```

Comment:
I know it's very nit picking but we use a code style (bracket on new line). maybe you can follow it?

Reply:
Yes, I have fixed the code style. Please check if it meets the requirement.